ptp/generate.go

```diff
@@ -149,7 +236,7 @@ func generateToolpath(argv []string) error {
 	if err = writer.Initialize(); err != nil {
 		return err
 	}
-
+	seen := false
```

Comment:
Looks like this variable was used for testing/isn't actually used for final logic?